lib/ActivityStream.jsm

```diff
@@ -247,6 +247,10 @@ const PREFS_CONFIG = new Map([
     skipBroadcast: true,
     value: "{}",
   }],
+  ["discoverystream.endpointSpocsClear", {
+    title: "Endpoint for when a user opts-out of sponsored content to delete the user's data from the ad server.",
+    value: "",
```

Comment:
When will we know what value to put here?

Reply:
Mathijs is setting that, he might be done, not sure.

In the end, there is a handful of turn it on related thing we need to do once all the pieces are ready and tested, so I'm guessing we have 1 patch either way to set that all up.